app_dart/lib/src/request_handlers/scheduler/gob_poller.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Is this change intended to be in this PR?

Reply:
Removed